### PR TITLE
Use ActionLauncher for LivermorePE, CombinedBrem, MuBremsstrahlung, RelativisticBrem

### PR DESCRIPTION
**Pull request by pcanal**

### CODE REVIEW

File: src/celeritas/em/model/LivermorePEModel.hh [hunk 1]
```diff
@@ -12,14 +12,16 @@
 #include "corecel/data/CollectionMirror.hh"
 #include "celeritas/Quantities.hh"
 #include "celeritas/em/data/LivermorePEData.hh"
-#include "celeritas/mat/MaterialParams.hh"
+//#include "celeritas/mat/MaterialParams.hh"
 #include "celeritas/phys/AtomicNumber.hh"
 #include "celeritas/phys/Model.hh"
-#include "celeritas/phys/ParticleParams.hh"
+//#include "celeritas/phys/ParticleParams.hh"
```
Comment (sethrj) [inline src/celeritas/em/model/LivermorePEModel.hh]: Remove the temporary comments

**Review by sethrj — COMMENTED** [review]
One comment, and can you run `clang-format -i` on the files you changed? I spotted a couple of small spacing issues.

**Comment by pcanal** [comment]
Indeed :( .. `[git]-clang-format` had silently stopped working (claimed success when clang-format actually failed to run)

File: src/celeritas/em/model/LivermorePEModel.hh [hunk 2]
```diff
@@ -12,14 +12,16 @@
 #include "corecel/data/CollectionMirror.hh"
 #include "celeritas/Quantities.hh"
 #include "celeritas/em/data/LivermorePEData.hh"
-#include "celeritas/mat/MaterialParams.hh"
+// #include "celeritas/mat/MaterialParams.hh"
 #include "celeritas/phys/AtomicNumber.hh"
 #include "celeritas/phys/Model.hh"
-#include "celeritas/phys/ParticleParams.hh"
+// #include "celeritas/phys/ParticleParams.hh"
```
Comment (sethrj) [inline src/celeritas/em/model/LivermorePEModel.hh]: This was marked as resolved earlier but these dead comments still need to be deleted

**Review by sethrj — APPROVED** [review]

**Pull request merged**